### DISCUSSION TITLE
Included Astro objects

### DISCUSSION THREAD

**Discussion by voetberg** [deepskies/DeepBench#90] [Ideas]
The out-of-date readme has recommendation for some objects that we're missing. 

For a beta release, which ones do we think would be good to add? 
Currently listed options are: 

- Supernovae: light curves
- Quasars: Skewed Sine wave
- N-body simulations: Points in 2D and 3D and in lightcones
- Galaxy clusters: SZ - blurred circles; X-ray - blurred circles
- Spectra (stellar, galactic)
- Noise: photon 

@beckynevin @omaripaul57 @bnord

**Comment by bnord** [comment]
Those recommendation objects were aspirational from a long long long long time ago.
If we're about to do alpha release, then I envision the following the following priority for Beta (if that would be later)
1. Light curves: up and down once for supernovae; skewed sine wave for quasars
2. Radial profiles: Gaussian with cut-off for matter density or light profiles of galaxy clusters (we might be able to do a prototype of this this summer via the cluster fast sim completions)
3. Spectral densities: straight and slanted lines with spikes and dips for things like quasars adn high-redshift galaxies; Tanh with spikes and dips for large red galaxies
4. N-body simulations: collection of points in 2d and 3d that have a distribution; evolve the distribution over time under some prescription like gravity (we might get a proto of this done this summer through another student project)
